opus/import/obs_instrument_hststis.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, *args, **kwargs):
 
 
     def _stis_spec_flag(self):
-        obs_type = self.field_obs_general_observation_type()
+        obs_type = self.field_obs_general_observation_type()['col_val']
```

Comment:
This looks kind of ugly now. Perhaps we need a function `_observation_type` (put right above `field_obs_general_observation_type`) that does the work of the old `field_obs_general_observation_type` and just returns the raw value. That could be used in this function. Then the full `field_obs_general_observation_type` could call `_observation_type` and turn it into a mult with no additional work needed.